csharp/ql/src/semmlecode.csharp.dbscheme

```diff
@@ -497,10 +497,6 @@ expr_flowstate(unique int id: @expr ref, int state: int ref);
 
 @generic = @type | @method | @local_function;
 
-is_generic(unique int id: @generic ref);
```

Comment:
Do we need a corresponding DBscheme hash bump on master for odasa?

Reply:
Yes, I will open an internal PR as well.